README.md

```diff
@@ -179,7 +197,7 @@ interface BlockDefinition {
   title: string;
   description: string;
   entry: string;
-  matches?: string[]; // An array of globs written in picomatch syntax. See https://github.com/micromatch/picomatch for examples.
+  matches?: string[];
```

Comment:
any reason you removed this comment? just curious

Reply:
just because it duplicates the explanation in the following paragraph